### DISCUSSION TITLE
Vercel doesn't cache node_modules folder

### DISCUSSION THREAD

**Discussion by ramiel** [vercel/vercel#7642] [Help]
I have a monorepo with a project being a serverless function, it juts contains an `/api/index.ts` file. When it builds on vercel, packages are downloaded and installed all the time. The project root folder is configured correctly and I also use `ENABLE_ROOT_PATH_BUILD_CACHE` env variable to `1`. The build cache never contains the node_modules folder    
![Screenshot_20220401_093601](https://user-images.githubusercontent.com/324687/161217189-332aca57-a07e-4393-b04a-208184fa2a87.png)

This results in slow build (>1m10s). Is there anything I can do to prevent this problem?

**Comment by BrettBedarf** [comment]
+1

**Comment by jcguarinpenaranda** [comment]
+1
